### PR TITLE
feat(form): add the form generator to the development site

### LINKED ISSUE #291: Wire schema-form-generator to active component on the Examples page
On the page with numerous examples, I should be able to select any example and make changes via the schema form generator.

### PR DESCRIPTION
**Pull request by janechu**
Closes #291

### CODE REVIEW

**Comment by janechu** [comment]
Just a note, I have removed a few dependencies that cause circular dependencies in our packages. For example we cant rely on fast-components-react-base because that is used by fast-components-react-msft and if the development site uses the form generator which uses the base, it will become circular.

File: packages/fast-development-site-react/src/components/site/action-bar.tsx [hunk 1]
```diff
@@ -15,30 +15,87 @@ export interface IActionBarProps extends RouteComponentProps<any> {
...
+    return {
+        position: "relative",
+        height: "40px",
+        border: "none",
+        background: "none",
+        padding: "0",
+        "& span": {
+            width: "16px",
+            height: "16px",
+            marginRight: "5px",
```
Comment (nalogiudice) [inline packages/fast-development-site-react/src/components/site/action-bar.tsx]: `toPx()` ?

File: packages/fast-development-site-react/src/components/site/component-wrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    ICSSRules,
+    IJSSManagerProps,
+    IManagedClasses
+} from "@microsoft/fast-jss-manager-react";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import devSiteDesignSystemDefaults, { IDevSiteDesignSystem } from "../design-system";
+
+export interface IComponentWrapperManagedClasses {
+    componentWrapper: string;
+    componentWrapper__active: string;
```
Comment (nalogiudice) [inline packages/fast-development-site-react/src/components/site/component-wrapper.tsx]: I thought we were only doing a single _ ?

File: packages/fast-development-site-react/src/components/site/action-bar.tsx (hunk 1 above)
Comment (janechu) [reply packages/fast-development-site-react/src/components/site/action-bar.tsx]: fixed

**Pull request merged**